components/google_workspace/actions/list-activities-by-admin/list-activities-by-admin.ts

```diff
@@ -1,3 +1,4 @@
+import "gaxios";
```

Comment:
Is this one used somewhere? It is not in `package.json`, if it is a dependency used can we add it to `package.json` too?

Reply:
I'll remove it. Thanks